extensions/amp-story/1.0/amp-story.css

```diff
@@ -946,3 +946,7 @@ amp-story-page .i-amphtml-story-page-open-attachment-label {
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+amp-story-page amp-twitter {
```

Comment:
`amp-story-grid-layer` only plz, to exclude tweets in `amp-story-page-attachment` :))

Reply:
Also, could you please add a comment to say where this 500px is from?

Reply:
Oh yeah. Done.